tests/test_charm.py

```diff
@@ -105,3 +109,107 @@ def test_new_unit_has_password(self):
             config["MYSQL_ROOT_PASSWORD"],
             self.harness.charm._stored.mysql_setup["MYSQL_ROOT_PASSWORD"],
         )
+
+    def test_provides(self):
```

Comment:
Just a stylistic suggestion, so you may choose to ignore if you do not like the suggestion. It will be great if tests names are a bit more descriptive. For example "test_version_and_db_info_is_provided" or anything else you think appropriate.

Reply:
Agree